packages/in_app_purchase/in_app_purchase_storekit/test/store_kit_wrappers/sk_product_test.dart

```diff
@@ -38,6 +38,16 @@ void main() {
       expect(wrapper, equals(dummyDiscount));
     });
 
+    test(
```

Comment:
can we also add a test (or modify a current test) that checks the correct identifier and type are created and passed?

Reply:
It has been included in the test on line 33